wine-tkg-git/wine-tkg-scripts/build.sh

```diff
@@ -121,38 +121,37 @@ _build_serial() {
   if [ "$_NOLIB64" != "true" ]; then
     # build wine 64-bit
     # (according to the wine wiki, this 64-bit/32-bit building order is mandatory)
+    if [[ "$_nomakepkg_dependency_autoresolver" == "true" ]]; then
+      install_deps "64" "${_ci_build}"
+    fi
     _exports_64
     _configure_64
     _build_64
   fi
   if [ "$_NOLIB32" != "true" ] && [ "$_NOLIB32" != "wow64" ]; then
     # build wine 32-bit
     # nomakepkg
-    if [ "$_nomakepkg_midbuild_prompt" = "true" ]; then
+    if [[ "$_nomakepkg_midbuild_prompt" == "true" ]]; then
       msg2 '64-bit side has been built, 32-bit will follow.'
       msg2 'This is the time to install the 32-bit devel packages you might need.'
       read -rp "    When ready, press enter to continue.."
     fi
-    if [ "$_nomakepkg_dep_resolution_distro" = "debuntu" ]; then
-      _debuntu_32
+    if [[ "$_nomakepkg_dependency_autoresolver" == "true" ]]; then
```

Comment:
We probably want to keep the style consistent and mimic the less bash'd line 146

Reply:
Resolved with [2c0b5e5](https://github.com/Frogging-Family/wine-tkg-git/pull/1138/commits/2c0b5e5bd364e3b90cc93a39636d95ce114b1868) and https://github.com/Frogging-Family/wine-tkg-git/pull/1138/commits/df0f5a7e5250add6be91e42ca0398df5d0409fd8